wiki/Tournaments/CFC/en.md

```diff
@@ -0,0 +1,238 @@
+---
+tags:
+- CFC 2018
+- CFC2018
+---
+# Catch French Cup 2018
+
+![Catch French Cup 2018](logo.png)
+
+**CFC 2018**, short for **Catch French Cup 2018** is a one-on-one french-based osu!catch tournament hosted and organized by ![][flag_FR] [Kasumii-sama](https://osu.ppy.sh/users/6177263).
+
+## Tournament Schedule
+
+| Event              | Timestamp                                  |
+|-------------------:|--------------------------------------------|
+| Registration Phase | 2018-03-11/2018-04-01 |
+| Group Stage        | 2018-04-07/2018-04-09 |
+| Quarterfinals + Loser's Bracket Round 1 | 2018-04-14/2018-04-15 |
+| Semifinals + Loser's Bracket Round 2 | 2018-04-21/2018-04-22  |
+| Finals - Week 1    | 2018-04-28/2018-04-29  |
+| Finals - Week 2    | 2018-05-05/2018-05-06 |
+
+## Prizes
+
+| Placing                                             | Prize(s)                             |
+|----------------------------------------------------|---------------------------------------|
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | Profile badge + 8 months of supporter |
```

Comment:
Center the crowns.

Reply:
i'm sorry, but i don't get how to do this one?

Reply:
ty, should be good now